algorithms/integration/kapton_2019_correction.py

```diff
@@ -242,6 +257,85 @@ def _check_int_edge_pts(int_edge_pts):
                     new_int_edge_pts[3] = int_edge_pts[1]
                 return new_int_edge_pts
 
+        def _orient_det_edge_pts(det_edge_pts):
```

Comment:
The original logic makes an unstated assumption that the detector edge points used to help calculate the location of the absorption max and edge (aka min) will be perpendicular to the y-axis. The original logic ran into edge cases where this was no longer true. This logic attempts to skirt this issue by enforcing the detector edge points to listed in a certain order based on location relative to the origin of the detector.